invenio_communities/views/communities.py

```diff
@@ -172,6 +194,14 @@ def invitations(community=None, pid_value=None):
             "topic": "Topic",
             "project": "Project"
         },
-        permissions=community.has_permissions_to(['update']),
+        permissions=permissions(community),
         active_menu_tab="members",
     )
+
+
+def permissions(community):
+    """Calculates permissions for the given community"""
+    permissions = {}
+    permissions.update(community.has_permissions_to(['update']))
+    permissions.update(community.has_permissions_to(['create']))
```

Comment:
why is create permission needed?

Reply:
As far as I understand, the users that can see and access the requests tab are the users that have the can_create permissions for the community:
can_create = [AuthenticatedUser(), SystemProcess()]

In order to hide the tab from user that don't have the can_create permission, as it was done for the can_update permission with the settings, this information must be passed to the jinja template.

Reply:
Actually, why are we touching permissions?